test/libsolidity/syntaxTests/inheritance/override/override_shared_base_simple.sol

```diff
@@ -0,0 +1,6 @@
+contract I {
+  function f() external {}
+}
+contract A is I {}
+contract B is I {}
+contract C is A, B { function f() external override {} }
```

Comment:
Ah, here is the test. So is the override really needed here?

Reply:
I mean is it required here to re-define `f`

Reply:
No it's not, I removed it now.